pkg/distro/fedora/images.go

```diff
@@ -223,6 +223,13 @@ func liveImage(workload workload.Workload,
 
 	img.Filename = t.Filename()
 
+	distro := t.Arch().Distro()
+
+	// We start marking packages in the dnf state database in Fedora 39 and up
+	if !common.VersionLessThan(distro.Releasever(), "39") {
+		img.MarkPackages = true
+	}
```

Comment:
Do we really need this? I think that we can consider not marking packages being a bug, no? I mean, if this can't break users, let's do it for all versions :thinking:

Reply:
Since it changes behavior of future images I'd prefer not to do it for current versions but I also don't hold that opinion very strongly. Enabling it on everything is fine by me.

Reply:
@ondrejbudai WDYT?

Reply:
I'm happy to treat it as a regression, so enable it also for older Fedoras.